src/main/java/frc/robot/commands/IntakeCommands/IntakePassNoteToPizzaBox.java

```diff
@@ -45,28 +45,31 @@ public void execute() {
   public void end(boolean interrupted) {
     pizzaBoxSubsystem.stopPizzaBoxMotor();
     intakeSubsystem.stopIntakeMotors();
+    if(interrupted) {
+      intakeSubsystem.rejectIntake(30, 100);
+    }
   }
 
   // Returns true when the command should end; called every cycle
   @Override
   public boolean isFinished() {
 
-    return false;
-    // prevSensorValue = currentSensorValue;
-    // currentSensorValue = intakeSubsystem.isNotePresent();
-
-    // if(prevSensorValue == true && currentSensorValue == false) {
-    //   noteLatch = true;
-    // }  
-    // // two second delay before checking sensor again
-    // if(counter > Constants.IntakeConstants.NOTE_DELAY && noteLatch) {
-    //   noteLatch = false;
-    //   return true; 
-    // }
-    // else {
-    //   counter++;
-    // }
     // return false;
+    prevSensorValue = currentSensorValue;
+    currentSensorValue = intakeSubsystem.isNotePresent();
+
+    if(prevSensorValue == true && currentSensorValue == false) {
+      noteLatch = true;
+    }  
+    // two second delay before checking sensor again
```

Comment:
instead of a counter, we can remember the start time, and do a difference based on clock time.